client/stylesheets/main.scss

```diff
@@ -1,3 +1 @@
-@import "vars";
-@import "toCat";
-// @import "normalize.css";
+@import "client/stylesheets/main.scss";
```

Comment:
Files should end with a trailing newline<br>Imported partial `client/stylesheets/main.scss` should be written as `client/stylesheets/main`